confmap/confmap.go

```diff
@@ -181,6 +181,15 @@ func (l *Conf) Merge(in *Conf) error {
 	return l.k.Merge(in.k)
 }
 
+// Delete a path from the Conf.
+// Return true if the path is deleted.
+// If the path is not set (so the operation is a no-op), it returns false.
```

Comment:
I think we can make this a bit clearer:
```suggestion
// If the path exists, deletes it and returns true.
// If the path does not exist, does nothing and returns false.
```